moto/core/utils.py

```diff
@@ -307,3 +307,15 @@ def extract_region_from_aws_authorization(string: str) -> Optional[str]:
     if region == auth:
         return None
     return region
+
+
+def params_sort_function(item: tuple[str, Any]) -> tuple[str, Any]:
```

Comment:
```suggestion
def params_sort_function(item: Tuple[str, Any]) -> Tuple[str, Any]:
```
MyPy has it's own tuple-type that can be used in these cases - that should fix the build